src/Altinn.FileScan/Services/AccessTokenService.cs

```diff
@@ -51,10 +51,8 @@ public async Task<string> Generate()
             string certBase64 =
                   await _keyVault.GetCertificateAsync(CertId);
 
-            accessToken = _accessTokenGenerator.GenerateAccessToken("platform", "file-scan", new X509Certificate2(
-                Convert.FromBase64String(certBase64),
-                (string)null,
-                X509KeyStorageFlags.MachineKeySet | X509KeyStorageFlags.PersistKeySet | X509KeyStorageFlags.Exportable));
+            var certificate = X509CertificateLoader.LoadPkcs12(Convert.FromBase64String(certBase64), (string)null, X509KeyStorageFlags.MachineKeySet | X509KeyStorageFlags.PersistKeySet | X509KeyStorageFlags.Exportable);
```

Comment:
Should we try to fix the implementation in PlatformKeyVaultService as well? Looks identical. Tries to access the same certificate.